vcf2fhir/fhir_helper.py

```diff
@@ -35,21 +35,40 @@ def __init__(self, patientID):
     def _get_region_studied_component(
             self,
             reportable_query_regions,
-            nocall_regions):
+            nocall_regions,
+            is_null):
+        ranges_examined_component = concept.CodeableConcept(
+            {
+                "coding": [
+                    {
+                        "system": "http://loinc.org",
+                        "code": "51959-5",
+                        "display": "Ranges-examined component"
+                    }
+                ]
+            }
+        )
+        if is_null:
+            observation_rs_component = observation.ObservationComponent()
+            observation_rs_component.code = ranges_examined_component
+            observation_rs_component\
+                .dataAbsentReason = concept.CodeableConcept(
+                    {
+                        "coding": [
+                            {
+                                "system": ("http://terminology.hl7.org/" +
+                                           "CodeSystem/data-absent-reason"),
+                                "code": "not-performed",
+                                "display": "Not Performed"
+                            }
+                        ]
+                    }
+                )
+            return [observation_rs_component]
         observation_rs_components = []
         for _, row in reportable_query_regions.df.iterrows():
             obv_comp = observation.ObservationComponent()
-            obv_comp.code = concept.CodeableConcept(
-                {
-                    "coding": [
-                        {
-                            "system": "http://loinc.org",
-                            "code": "51959-5",
-                            "display": "Ranges-examined component"
-                        }
-                    ]

```

Comment:
What would this return for call [here](https://github.com/elimuinformatics/vcf2fhir/pull/69/files#diff-c137f67a35c45349de55155fe04f886d8c50d73952104ca2efe1f191c95d6aeaR138-R139)? Would it add anything to output `fhir` JSON? If not, I would move the below logic in this method instead of calling this method again towards the end. Moving this logic would require a new parameter `conversion_region` to the method signature, which is okay.
```
 distinct_chromosomes_cr =\ 
           conversion_region.as_df()['Chromosome'].drop_duplicates() 
       for chromosome in distinct_chromosomes_cr:
```

Reply:
where did this logic go ? Was it not required ?